### ISSUE TITLE
plaintext-node contains <rich_text> ... </rich_text> (xml saved in directory structure)

### ISSUE THREAD

**Issue by klartext** [giuspen/cherrytree#2573]
cherrytree 1.1.4
Arch-Linux

**Describe the bug**
A plaintext node has the text wrapped into `<rich_text>` and `</rich_text>`

**To Reproduce**
1) Create a plain text node, save the Tree as multiple files in hierarchical folder structure.

2) Then  look what's in the file.

**Note**
So far this has caused no problems in working with the file.
Nevertheless this looks like an inconsitency, which may cause problems in certain situations.

**Comment by giuspen** [comment]
That's right, currently even the plain text is inside a rich text capable slot, just without any tag applied.
So it is as expected.
I can consider removing the wrap before saving to file if that would be useful to cross edit with another text editor but that has to be done only while cherrytree is not running as monitoring the file for changes is not currently something cherrytree does in the multifile (it can do so if enabled in the preferences dialog for the single file document type).

**Comment by klartext** [comment]
Removing the XML-wrapping completely (not just replacing the `rich_text` tag with a `plain_text` tag or similar things), would be great.
If it's just a plain text file (assuming utf-8 here) that is used for plain text documents, that would allow painless editing of the files - without the fear of messing things up

And as you assumed correctly, I think about editing files directly with my favourite editor.
If cherrytree would allow starting the favourite text editor, that would be nice, but if it does not, it's also ok.
For versioning I use just git on the directory that contains the info-trees.

**Comment by klartext** [comment]
Do you have any time plan on removing the XML-wrapping for plain text files?
Is there a roadmap for cherrytree, or a priority list?

**Comment by giuspen** [comment]
I will try to do this together with https://github.com/giuspen/cherrytree/issues/2354
I have in my notes my planning but since this happens in my spare time I don't want to make that public as cannot guarantee
